ecosystem/python/sdk/aptos_sdk/client.py

```diff
@@ -632,3 +666,7 @@ def __init__(self, message, status_code):
         # Call the base class constructor with the parameters it needs
         super().__init__(message)
         self.status_code = status_code
+
+
+class ResourceNotFound(Exception):
+    """Errow thrown when the resource is not found"""
```

Comment:
"Error". Also generally, it's better to describe the error.  E.g. the resource was not found.

Reply:
rofl, copypasta from the previous error... though agree :)